homeassistant/const.py

```diff
@@ -917,6 +917,9 @@ class UnitOfPrecipitationDepth(StrEnum):
 CONCENTRATION_PARTS_PER_MILLION: Final = "ppm"
 CONCENTRATION_PARTS_PER_BILLION: Final = "ppb"
 
+# Pool chlorine unit
+CHLORINE_UNIT: Final = "mg/L"
```

Comment:
```suggestion
CONCENTRATION_MILLIGRAMS_PER_LITER: Final = "mg/L"
```
Why not `CONCENTRATION_MILLIGRAMS_PER_LITER`? The unit `mg/L` is not a unit solely for chlorine concentration.

Reply:
I did not know how to name it.
Corrected.

Reply:
All good! Thank you for the contribution.